auto_py_to_exe/utils.py

```diff
@@ -39,10 +40,35 @@ def open_output_in_explorer(output_directory, input_filename, is_one_file):
         return False
     return True
 
+def get_latest_pyinstaller_version():
+    """Check the latest version of PyInstaller"""
+    try:
+        response = requests.get("https://api.github.com/repos/pyinstaller/pyinstaller/releases/latest")
+        response.raise_for_status()
+        latest_version = response.json()["tag_name"]
+        return latest_version.strip('v')
+    except requests.exceptions.RequestException:
+        return None
 
 def get_warnings():
     warnings = []
 
+    # Check pyinstaller version is it latest
+    try:
+        latest_pyinstaller_version = get_latest_pyinstaller_version()
```

Comment:
Now that `latest_version` has been renamed to `latest_pyinstaller_version`, you need to update the other references. Currently, these changes will fail to run.